### ISSUE TITLE
Change is predictions for BERT-QA when data is shuffled

### ISSUE THREAD

**Issue by soumyasj** [ThilinaRajapakse/simpletransformers#1530]
I am trying to finetune BERT-QA. When I finetune and test the finetuned model, the results change if I simply shuffle the data.
Is there anything wrong with shuffling? Should I consider changing any parameters to the model?

**Comment by DamithDR** [comment]
Did you mean shuffling the training data or test data ?

**Comment by soumyasj** [comment]
Shuffling the test data. When I installed the simpletransformers library again it started working for the shuffled data too. I think there was some issue with the environment. I will be marking the issue closed as it is resolved.
